nservicebus/messaging/delayed-delivery.md

```diff
@@ -13,6 +13,16 @@ The message doesn't have to be dispatched immediately after sending, it can be d
 
 NOTE: Only send operations can be deferred. Publish and reply operations cannot be deferred.
 
+## Handling current message later
+To handle the current message at a later time `bus.HandleCurrentMessageLater()` method can be used.  This was the initial way of *deferring* a message. This would create a copy of the message that has the same identifier, header and body, and then put the new message at the end of the queue. The message eventually will be pick up again once other messages in the queue are processed. 
+
+There are two caveats with this method:
+
+- As mentioned above a new copy of the message will be put back at the end of the queue. To make this work, the message pipeline will not abort which means any business transaction will also get comitted. 
```

Comment:
> As mentioned above a new copy of the message will be put back at the end of the queue.

is this true for all transports?

> To make this work, the message pipeline will not abort which means any business transaction will also get comitted. 

it's probably worth mentioning that the pipeline doesn't invoke remaining message handlers when this method has been called.